### DISCUSSION TITLE
Import from ZKN (Zettelkasten(

### DISCUSSION THREAD

**Discussion by luzcosta** [Zettlr/Zettlr#5119] [Q&A]
I've been a zkn user since at least 2009, a great program,  when it works. The issue is that it has no development in terms of updates. Right now, I purchased a Mac (Macbook Air M1 with Sonoma 14.4.1) and I can't install it, I believe, because the version of Java that I think I have installed because I don't even know if it is installed, doesn't work, it doesn't open zkn well and when you open something similar to zkn the program doesn't close and a few more problems... Therefore, I am thinking about using another program and I would like to know a) if Zettlr is similar, b) if there is any type of support that is not for technicians and computer and code experts, c) especially if I can transfer my files from zkn for Zettlr. Thanks

**Comment by nathanlesage** [comment]
I myself used zkn3 quite some time before starting development of Zettlr, so it should work fine (with changes in workflow, obviously)
